packages/app/.gitignore

```diff
@@ -16,3 +16,6 @@ node_modules
 # vercel
 .vercel
 next-env.d.ts
+
+# nook
+/.nook
```

Comment:
If this is the folder you use to generate the output files when nook is gathering the data – I suggest to put it in the node_modules. There we can predictable import it with a regular require and users won't need to add it to their .gitignore